bin/all_sky_search/pycbc_fit_sngls_over_multiparam

```diff
@@ -62,6 +118,14 @@ parser.add_argument("--smoothing-width", type=float, nargs='+', required=True,
                          "logs of them) to smooth over. Required. "
                          "This must be a list corresponding to the smoothing "
                          "parameters.")
+parser.add_argument("--smoothing-method", default="smooth_area",
+                    choices = _smooth_dist_func.keys(),
+                    help="Method used to smooth the fit parameters; "
+                         "'smooth_area' (default) finds all templates within "
+                         "unity distance from the template of interest "
```

Comment:
-> 'unit' distance I think - you also should mention the 'weighted' method in this help string